bitnami/external-dns/README.md

```diff
@@ -70,6 +70,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `extraDeploy`           | Array of extra objects to deploy with the release (evaluated as a template).                 | `[]`            |
 | `kubeVersion`           | Force target Kubernetes version (using Helm capabilities if not set)                         | `""`            |
 | `watchReleaseNamespace` | Watch only namepsace used for the release                                                    | `false`         |
+| `kind`                  | Use a DaemonSet or Deployment                                                                | `Deployment`    |
```

Comment:
Other charts normally use `useDaemonset` as a boolean variable. Could you update the PR taking this change into account?